pages/blog/async-api-spec-3.0-release.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "AsyncAPI Spec 3.0 is coming!"
+date: 2022-02-28T06:00:00+01:00
+type: Communication
+tags:
+  - Specification
+  - Announcement
+  - Release
+cover: /img/posts/async-api-spec3.0-release/cover.webp
+authors:
+  - name: Harsh Mishra
+    photo: /img/avatars/harsh.webp
+    link: https://www.linkedin.com/in/harshcasper/
+excerpt: 'Announcing the start of work on AsyncAPI Spec 3.0. Checkout all the major new features coming to AsyncAPI and how you can get involved!'
```

Comment:
```suggestion
excerpt: 'Announcing the start of work on AsyncAPI Spec 3.0. Check out all the potential new features coming to AsyncAPI 3.0 and how you can get involved!'
```
It is more potential new features than actual features, at least for now. Not sure if we want to make that distingshion?